### DISCUSSION TITLE
need tutorial on how to use trigger

### DISCUSSION THREAD

**Discussion by noobiessss** [Nain57/Smart-AutoClicker#511] [General]
need tutorial on how to use trigger

**Comment by Tallavn** [comment]
i need too. i can't understand how it work

**Comment by robiau** [comment]
ive been looking for a indepth guide on how all this works with triggers and how to use counters i would appreciate a guide or video on how to use it . great program though just not enough info to use its max potential

**Reply by soapyli** [reply]
For instance: I am testing my scripts and tuning the params so as to let it run as fast as possible. The best way is to check the overall time ticks after some several times, say, 3, or 10, since the elapsed time for just 1 time is too short to measure.
So you can just add the counter each time. As the counter reaches 3 or 10, you can disable all the scenarios. Here you must use trigger and counter, and you are sure not to miss it!
Good luck!

**Comment by dvdspelert** [comment]
Same, just a hello world, trigger by phone call / notification (from other app) would get me started. Now can not use the (looking great!) app..

I'd like to let Automate create a notification that triggers a more complex automation :)

**Reply by hatbrox** [reply]
what you can do is copy the wiki from this site, convert it into MD format and create a skill in your AI tool (ChatGPT, Claude etc)
then explain your scenario (that's the hardest part, by far: describe correctly what you need)
your AI will then tell you what to do with Klick'R

I have exported my click'r scenario as a ZIP file and uploaded it in my AI tool and it help me:
create visual diagrams
create text tutorial
analyse potential issues (that's a bit harder, needs to teach AI until it gets it right)
